htmresearch/algorithms/simple_union_pooler.py

```diff
@@ -64,21 +68,27 @@ def updateHistory(self, activeCells):
       self._activeCellsHistory.pop(0)
```

Comment:
B/c we're popping from the front of a list it would be more efficient to use a deque for `_activeCellsHistory`, but out of scope for the PR so I can followup with the changes later.